pyproject.toml

```diff
@@ -59,3 +59,6 @@ exclude = ["tests*", "benchmarks", "examples"]
 
 [tool.hatch]
 version.source = "vcs"
+
+[tool.hatch.build.hooks.vcs]
```

Comment:
This is the wrong path, it should be "eventlet/_version.py".

Reply:
Updated and successfully tested locally. Good catch, thanks.